website/project/model.py

```diff
@@ -860,6 +860,7 @@ class Node(GuidStoredObject, AddonModelMixin, IdentifierMixin):
     primary_institution = fields.ForeignField('institution')
 
     affiliated_institutions = fields.ForeignField('institution', list=True)
+    visit = fields.IntegerField(default=0)
```

Comment:
If a conference tag is removed from a project, that project won't even be showing up in the meeting's page. To collect the visit count of that is meaningless then.